src/OrbitClientGgp/include/OrbitClientGgp/ClientGgp.h

```diff
@@ -80,6 +80,7 @@ class ClientGgp final : public CaptureListener {
   std::shared_ptr<grpc::Channel> grpc_channel_;
   ProcessData target_process_;
   orbit_client_data::ModuleManager module_manager_;
+  std::atomic<uint64_t> next_function_id_;
```

Comment:
Can you add a comment here? What is the next function representing?

Reply:
This is actually not used - removed - also fixed the id assignment code to use sequential id instead of address.